app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V1::OrdersController < ApplicationController
+  require_dependency "#{Rails.root}/app/validators/orders_controller"
```

Comment:
This was added to resolve issues with the module load ordering in development. The full path was needed in order to properly load in the validator class .

I have tried resolving the module load issue by referencing the given class like such `::Validators::OrdersController`, but that didn't resolve the issue.